### ISSUE TITLE
Register eventstore as a 'service'

### ISSUE THREAD

**Issue by frederiksf** [EventStore/EventStore#1581]
I would like to start multiple EventStore s as a service on a Ubuntu machine. How can i achieve that? P.s. I'm using the Community Edition.

**Comment by shaan1337** [comment]
Hello @frederiksf 

You can install EventStore from packagecloud :
https://packagecloud.io/EventStore/EventStore-OSS/install

This installs EventStore as a systemd service and you can control it with `systemctl [start-stop] eventstore.service`

**Comment by frederiksf** [comment]
@shaan1337 Thank you! I also found this [page](https://eventstore.org/docs/server/installing-from-debian-repositories/). Am i limited to one EventStore instance or how can i configure more than one?

**Comment by shaan1337** [comment]
@frederiksf yes, running EventStore as a service is currently limited to 1 instance per machine. You could try using [chroot environments](https://help.ubuntu.com/community/BasicChroot) to run multiple instances which should work in theory but it has not been tested internally.

**Comment by jen20** [comment]
This isn't strictly true: you could also run multiple copies by creating a template unit as described in [1][1], but that will require modification of the package contents. It's perhaps something we could look at for development purposes and to make multi-tenant installations more straightforward, however running multiple nodes _from the same cluster_ on a particular machine is not advisable.

[1]: https://www.digitalocean.com/community/tutorials/understanding-systemd-units-and-unit-files#creating-instance-units-from-template-unit-files
